### PR TITLE
Add json with mitigated images

### PR DESCRIPTION
**Pull request by tobiscr**
PR includes JSON with mitigated non-trusted images

### CODE REVIEW

**Review by dekiel — APPROVED** [review]

File: docs/security/imagemitigation.json [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+      "openSSF": {
+        "score": 5.6,
+        "review": {
+          "approvedImage": false,
+          "reason": ""
+        }
+      }
+    },
+    {
+      "registry": "docker.io/prom/*",
```
Comment (TorstenD-SAP) [inline docs/security/imagemitigation.json:23]: The registry as well as the source code link points to a provider and not to specific images / repos. To analyze the risk introduced by a dedicated FOSS component, the absolute link to the image / repo is needed.

**Review by TorstenD-SAP — CHANGES_REQUESTED** [review]
As far as I remember, the workflow to approve images starts with the tags give to an images on Docker hub and the openSSF scorecard score. If those values are not there / lower than a threshold, an extended analysis is necessary. I recommend to structure the json in the following way:

```json
	{
		"registry": "gcr.io/k8s-prow/*",
		"purpose" : "Prow CI",
		"publisher": "Prow community",
		"sourceCode": "https://github.com/kubernetes/test-infra/tree/master/prow",
		"docker": {
			"verified publisher": true/false
			"official image": true/false
		}	
		"openSSF score": 5.6,
		"manual analysis": {
			"contributors": 1100,
			"activity": {
				"frequency": "high",
				"evidence": "https://github.com/kubernetes/test-infra/commits/master/prow"
			}
		},
		"comment": "",
			"review": {
			"approvedImage": false,
			"reason": ""
        }
    }
```

The `manual analysis` section can be optional but need probably more criteria.

File: docs/security/imagemitigation.json (hunk 1 above)
Comment (tobiscr) [reply docs/security/imagemitigation.json:23]: Let me fix that ;)

Comment (tobiscr) [reply docs/security/imagemitigation.json:23]: QQ... is this also required for the prow images?

**Review by IwonaLanger — APPROVED** [review]

**Review by TorstenD-SAP — APPROVED** [review]
Lgtm

**Review by Sawthis — APPROVED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**